frappe/public/js/frappe/widgets/utils.js

```diff
@@ -163,4 +163,13 @@ function get_number_system(country) {
 	return number_system_map[country];
 }
 
-export { generate_route, generate_grid, build_summary_item, shorten_number };
+const map_defaults = {
+	center: [19.0800, 72.8961],
+	zoom: 13,
+	tiles: 'https://{s}.tile.openstreetmap.org/{z}/{x}/{y}.png',
+	options: {
+		attribution: '&copy; <a href="http://osm.org/copyright">OpenStreetMap</a> contributors'
+	}
+};
```

Comment:
Can this be moved to the frappe/frappe/public/js/frappe/utils/utils.js file instead?

Reply:
The code you mentioned here is outdated (currently added to the utils object instead of exporting the constant), but generally speaking yes.
I moved the constants to `frappe/frappe/public/js/frappe/utils/utils.js`